src/TraceEvent/NativeDlls.cs

```diff
@@ -24,7 +24,7 @@ public static void LoadNative(string simpleName)
         // When TraceEvent is loaded as embedded assembly the manifest path is <Unknown>
         // We use as fallback in that case the process executable location to enable scenarios where TraceEvent and related dlls
         // are loaded from byte arrays into the AppDomain to create self contained executables with no other dependent libraries. 
-        string assemblyLocation = Assembly.GetExecutingAssembly().ManifestModule.FullyQualifiedName;
+        string assemblyLocation = typeof(NativeDlls).AssemblyQualifiedName;
```

Comment:
This is not correct.  The AssemblyQualifedName is aa name like 'mscorlib, Version=4.0.0.0' and we want a file path name.   However you can fix this simply by typeof(NativeDlls).Assembly.ManifestModule.FullyQualifiedName